src/main/java/mate/academy/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.bookshop.exception;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.bookshop.dto.ResponseErrorDto;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(

```

Comment:
we can also add handling of the `EntityNotFoundException`, `DataProcessingException`(if you have it) and the one which will be thrown at failed ISBN uniqueness validation